tests/utils-trait.php

```diff
@@ -256,4 +256,20 @@ public function _allowIframes( $allowed, $context ) {
 
 		return $allowed;
 	}
+
+	/**
+	 * This behaviour is working as expected on the front end, but during tests we need to set this custom values because they are hidden in the UI.
+	 * It was working before because PHP 7.3 was hiding the warning for access to an inexistent array key.
+	 * In the past if you try to read $options['key'] even $option was not an array no warning was triggered.
+	 */
+	public function _setPdfOptionsForTesting() {
```

Comment:
Setting default values for PDF exports during tests (before PHP 7.4) no warnings were triggered but new versions are more strict, those values does not affect the behaviour just needs to be present for some conditionals